saleor/account/i18n.py

```diff
@@ -191,6 +193,19 @@ def clean(self):
         data = super().clean()
         return self.validate_address(data)
 
+    @staticmethod
+    def substitute_invalid_values(data):
+        """Map invalid address names to the values accepted by i18naddress package."""
+        country_code = data["country_code"]
+        if not country_code:
+            return
+        if custom_names := VALID_ADDRESS_EXTENSION_MAP.get(country_code):
+            for field_name, mapping in custom_names.items():
+                actual_value = data.get(field_name, "").strip().lower()
+                mapping_adjusted = {k.strip().lower(): v for k, v in mapping.items()}
```

Comment:
Correct me if I am wrong, but `mapping` is the value that you fetched from `VALID_ADDRESS_EXTENSION_MAP`. Do we need to always make `strip().lower()` for the map declared by us? Can't we make this once, when initializing the map, instead of doing it always when validating any address?